compute_endpoint/globus_compute_endpoint/version.py

```diff
@@ -1,6 +1,6 @@
 # single source of truth for package version,
 # see https://packaging.python.org/en/latest/single_source_version/
-__version__ = "2.3.3a0"
+__version__ = "2.3.3"
```

Comment:
It seems that we should bump to `2.4.0` since this release adds new features.

Reply:
Yes, makes sense.  I have to rename the branch might as well rename the PR.  Closing this one.